### PR TITLE
Use payload types map of endpoint to which track belongs

### PR DESCRIPTION
**Pull request by mstyura**
It seems that this change fixes an issue (or not an issue, I'm not sure) described here #1163.

### CODE REVIEW

**Comment by mstyura** [comment]
Jenkins test this please.

File: src/main/java/org/jitsi/videobridge/cc/BitrateController.java [hunk 1]
```diff
@@ -771,14 +771,21 @@ private synchronized void update()
             // creating local final variables and pass that to the lambda function
             // in order to avoid that.
             final String endpointID = trackBitrateAllocation.endpointID;
+
+            final Endpoint endpointOwningTrack = destinationEndpoint.getConference().getLocalEndpoint(endpointID);
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/cc/BitrateController.java:775]: We'll have to discuss if we want to fix this or not (as you've seen, quite a few things assuming the same payload types), but if we do do this, we should expose the payload types a different way, I think.  The first thing that comes to mind is having the `Transceiver` expose a read-only version of its `StreamInformationStore` (which we could then expose at the `Endpoint` level)

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/cc/BitrateController.java:775]: See https://github.com/jitsi/jitsi-media-transform/blob/master/src/main/kotlin/org/jitsi/nlj/Transceiver.kt#L75

**Comment by mstyura** [comment]
@bbaldino sorry for bothering you, did you have a chance to discuss this change in a team? So far I don't see any problem with this change: running native Android/iOS WebRTC in same conference with Chromium's WebRTC and Firefox's WebRTC. As long as for jitsi meed use-case where all endpoints have same payload types assignment this change is actual no-op and maybe this PR can be safely merged? Thank you very much!

**Comment by mstyura** [comment]
Seems like the scenario I've tried to fix is not valid according to [BUNDLE spec](https://tools.ietf.org/html/draft-ietf-mmusic-sdp-bundle-negotiation-54#section-9.1.1):
```
9.1.1.  Payload Type (PT) Value Reuse

   Multiple bundled "m=" sections might describe RTP based media.  As
   all RTP based media associated with a BUNDLE group belong to the same
   RTP session, in order for a given payload type value to be used
   inside more than one bundled "m=" section, all codecs associated with
   the payload type number MUST share an identical codec configuration.
   This means that the codecs MUST share the same media type, encoding
   name, clock rate and any parameter that can affect the codec
   configuration and packetization.
   [I-D.ietf-mmusic-sdp-mux-attributes] lists SDP attributes, whose
   attribute values are required to be identical for all codecs that use
   the same payload type value.
```
So all endpoints in same conference should have same payload type assignment.

**Pull request closed without merging**